Dockerfile

```diff
@@ -90,131 +59,25 @@ RUN gnuArch="$(dpkg-architecture --query DEB_HOST_ARCH_CPU)"\
   && curl -SfL https://archive.debian.org/debian/pool/main/g/gcc-6/libgfortran3_6.3.0-18+deb9u1_${gnuArch}.deb -o libgfortran3.deb
 
 FROM python:${PYTHON_VERSION}-slim-${DEBIAN_VERSION} AS energyplus-dependencies
-ARG OPENSTUDIO_VERSION=3.8.0
-ARG OPENSTUDIO_VERSION_SHA=f953b6fcaf
-ARG ENERGYPLUS_VERSION=24.1.0
-ARG ENERGYPLUS_VERSION_SHA=9d7789a3ac
+ARG OPENSTUDIO_VERSION=3.9.0
+ARG OPENSTUDIO_VERSION_SHA=c77fbb9569
+ARG ENERGYPLUS_VERSION=24.2.0
+ARG ENERGYPLUS_VERSION_SHA=94a887817b
 
 RUN apt-get update \
   && apt-get install -y \
   curl \
+  binutils \
   && rm -rf /var/lib/apt/lists/*
 
 WORKDIR /artifacts
 
-RUN export gnuArch=x86_64; if [ "$(uname -m)" = "aarch64" ]; then gnuArch=arm64; fi; export gnuArch \
-  && echo https://github.com/NREL/EnergyPlus/releases/download/v${ENERGYPLUS_VERSION}/EnergyPlus-${ENERGYPLUS_VERSION}-${ENERGYPLUS_VERSION_SHA}-Linux-Ubuntu22.04-${gnuArch}.tar.gz \
-  && curl -SfL https://github.com/NREL/EnergyPlus/releases/download/v${ENERGYPLUS_VERSION}/EnergyPlus-${ENERGYPLUS_VERSION}-${ENERGYPLUS_VERSION_SHA}-Linux-Ubuntu22.04-${gnuArch}.tar.gz -o energyplus.tar.gz \
-  && curl -SfL https://github.com/NREL/OpenStudio/releases/download/v${OPENSTUDIO_VERSION}/OpenStudio-${OPENSTUDIO_VERSION}+${OPENSTUDIO_VERSION_SHA}-Ubuntu-22.04-${gnuArch}.deb -o openstudio.deb \
-  && curl -SfL https://openstudio-resources.s3.amazonaws.com/bcvtb-linux.tar.gz -o bcvtb.tar.gz
-
-FROM python:${PYTHON_VERSION}-slim-${DEBIAN_VERSION} AS dual-python
+RUN set -eux; \
+  export gnuArch=x86_64; if [ "$(uname -m)" = "aarch64" ]; then gnuArch=arm64; fi; export gnuArch; \
+  curl -SfL https://github.com/NREL/EnergyPlus/releases/download/v${ENERGYPLUS_VERSION}a/EnergyPlus-${ENERGYPLUS_VERSION}-${ENERGYPLUS_VERSION_SHA}-Linux-Ubuntu22.04-${gnuArch}.tar.gz -o energyplus.tar.gz; \
+  curl -SfL https://github.com/NREL/OpenStudio/releases/download/v${OPENSTUDIO_VERSION}/OpenStudio-${OPENSTUDIO_VERSION}+${OPENSTUDIO_VERSION_SHA}-Ubuntu-22.04-${gnuArch}.deb -o openstudio.deb
 
-  ENV PYTHON_VERSION=3.8.19
```

Comment:
Since we're no longer using two different python versions we don't need this build step